packages/next/taskfile.js

```diff
@@ -6,18 +6,24 @@ const babelClientOpts = {
     '@babel/preset-typescript',
     ['@babel/preset-env', {
       modules: 'commonjs',
-      targets: {
-        'esmodules': true
-      },
+      targets: 'edge 16, firefox 60, chrome 61, safari 11, opera 48, ios_saf 11, and_ff 64',
```

Comment:
This looks wrong 🤔 cc @keanulee

Reply:
Yeah, the PR is still a draft, and I've been trying a few things to get the bundle sizes under control.  

The `esmodules` target inadvertently became very broad in the latest Babel, largely due to a bunch of other transformations being turned on for Safari 10 workarounds.

Reply:
Targeting newer browsers here (instead of `esmodules`) seems fine - those Safari 10 workarounds (TIL) should still be provided when building with the default Babel config.

Reply:
I'm probably going to take this out of draft in a few days when I have another chance to test everything out.  

I think there are a few issues at play, but I think we may need to just accept larger bundle sizes if we want to upgrade Babel (or wait for their next point-release to see if they manage to get things back under control).